packages/babel-preset-stage-3/index.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   plugins: [
-    require("babel-plugin-transform-async-to-generator"),
-    require("babel-plugin-transform-exponentiation-operator")
```

Comment:
Sorry, we don't update the version manually in package.json and use https://github.com/lerna/lerna